testgrid/cmd/configurator/config_test.go

```diff
@@ -321,11 +321,17 @@ func TestConfig(t *testing.T) {
 	}
 
 	// All Testgroup should be mapped to one or more tabs
+	missedTestgroups := false
 	for testgroupname, occurrence := range testgroupMap {
 		if occurrence == 1 {
-			t.Errorf("Testgroup %v - defined but not used in any dashboards", testgroupname)
+			t.Errorf("Testgroup %v - defined but not used in any dashboards.", testgroupname)
+			missedTestgroups = true
 		}
 	}
+	if missedTestgroups {
+		t.Logf("Note: Testgroups are automatically defined for postsubmits and periodics.")
```

Comment:
Would like to eventually do it for presubmits too.

Reply:
Agree! We are a bit sketchy on our enforcement of that today, so I stuck to the easy ones.